pkgs/top-level/all-packages.nix

```diff
@@ -19700,8 +19700,12 @@ with pkgs;
     sdk = true;
   };
 
+  nrf-command-line-tools = callPackage ../development/tools/misc/nrf-command-line-tools { };
```

Comment:
I think these maybe should go to `pkgs/by-name` now.

Reply:
Yeah, I think so. There's [RFC 140](https://github.com/NixOS/rfcs/pull/140) that details the move, but there's a shorter description at [`pkgs/by-name/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md). I think after #275539, this should've become mandatory for new packages, but the pipeline might not have triggered on it since the last push was before this was merged.

Reply:
Okay, thank you. I have moved the packages to their new location.